<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="main_grid_layout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>15</number>
    </property>
    <item row="3" column="2">
     <widget class="QPushButton" name="pause_switch">
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Switch</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="browse_button">
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="screen_define_switch">
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Switch</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="image_extension_input">
      <property name="maximumSize">
       <size>
        <width>283</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="image_name_input">
      <property name="maximumSize">
       <size>
        <width>283</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="path_input">
      <property name="maximumSize">
       <size>
        <width>283</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>83</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Folder to save</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Screen after defining</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>83</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Image name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Image extension</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_13">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>83</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Paused</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Screenshot backend</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="paused_label">
      <property name="text">
       <string>False</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="screen_definin_label">
      <property name="text">
       <string>True</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="backend_choice">
      <item>
       <property name="text">
        <string>mss</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pil</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>scrot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>gnome-screenshot</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="last_layout" stretch="3,6,3">
    <item>
     <widget class="QPushButton" name="confirm_button">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
